operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -90,47 +122,55 @@ A DNS SRV record is structured as follows:
 - Name: The domain name for which this record is valid.
 - Other elements were omitted for clarity and brevity.
 
+```text
+<service>.<protocol>.<domain name>
+```
+
 For example, DNS SRV record might look like this:
 
 ```text
-_sbps-hsn._tcp.my-system.my-site-domain.net
+_sbps-hsn._tcp.my-system.my-site-domain
 ```
 
 The following is an explanation of the values used in this example:
 
 - `_sbps-hsn` is the symbolic name of the service.
+  - In this example, it is using the High Speed Network (hsn).
```

Comment:
This can be reworded as "In this example, High Speed Network (hsn) is used"

Reply:
Fixed.